system/jlib/jplane.cpp

```diff
@@ -231,6 +233,39 @@ class CStoragePlane final : public CInterfaceOf<IStoragePlane>
         return hosts;
     }
 
+    virtual bool isLocalPlane() const override
+    {
+        CriticalBlock b(isLocalCrit);
+        if (cachedLocalPlane)
+            return isLocal;
+
+        isLocal = false;
+        if (hosts.size()== 0)
+        {
+            isLocal = true;
+        }
+        else
+        {
+            const char * myHostName = GetCachedHostName();
+            for (auto &host: hosts)
+            {
+                if (strsame(host.c_str(), ".") || strsame(host.c_str(), "localhost"))
```

Comment:
can a plane ever have "." in it?
It feels like if that ever happens (via environmnet BM creation), that it should be translated to "localhost"